docs/src/getting-started.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started
+
+## For Users
+
+### Installation
+
+First, download and install Julia by following the instructions at [https://julialang.org/downloads/](https://julialang.org/downloads/).
+Then, you can install the ClimaLand package by doing:
+
+```julia
+julia> ] # Enter Package REPL mode
+Pkg> add ClimaLand # Install ClimaLand
+Pkg> # Go back to Julia REPL mode
+Julia> using ClimaLand
+```
+
```

Comment:
The transition from "Installation" to "Parametrisation" feels abrupt. 

Let's add a transition, like:
```
A typical land simulation employs several different parameterizations to model the various land-surface processes. Let's start our journet into ClimaLand by looking at one of those.
```

Reply:
Thank you!! I like that